src/blurbehind.cc

```diff
@@ -1,14 +1,13 @@
+#include <nan.h>
 #include <dwmapi.h>
-#include <node.h>
-#include <node_buffer.h>
 
 void blurbehind(const v8::FunctionCallbackInfo<v8::Value> &args)
 {
 	const HWND handle = *reinterpret_cast<HWND *>(node::Buffer::Data(args[0].As<v8::Object>()));
 
 	const DWM_BLURBEHIND bb = {
 		DWM_BB_ENABLE,
-		args[1]->BooleanValue(),
+		args[1]->BooleanValue(Nan::GetCurrentContext()).FromJust(),
```

Comment:
Why bring in the whole of Nan instead of just calling `v8::Isolate::GetCurrent()->GetCurrentContext()` which has shown to be stable across many versions of v8?

Also, to keep consistent with `windows-swca` and avoid this whole context thing while still keeping compatibility with a wide range of versions, I would prefer a call to `.As<v8::Boolean>()->Value()`

Reply:
Done!